test/integration/admin_can_login_to_admin_dashboard_test.rb

```diff
@@ -0,0 +1,20 @@
+require "test_helper"
+
+class AdminCanLoginToAdminDashboardTest < ActionDispatch::IntegrationTest
+  test "admins logs in and sees admin dashboard" do
+    admin = User.create(first_name: 'Admin',
+                     last_name: 'Admin',
```

Comment:
Align the elements of a hash literal if they span more than one line.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.